scripts/one-chain

```diff
@@ -133,5 +133,7 @@ $BINARY --home $CHAIN_DIR/$CHAIN_ID start --pruning=nothing --grpc.address="0.0.
 # Show validator's and user's balance
 sleep 3
 RPC_ADDR="tcp://localhost:$RPC_PORT"
+echo "Balances for validator '$VALIDATOR' @ '$RPC_ADDR'"
```

Comment:
Made it more explicit as it wasn't clear from output which was user <> validator.